ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ibgp/neighbors.j2

```diff
@@ -37,3 +37,9 @@
       description: {{ evpn_route_client }}
 {%     endfor %}
 {% endif %}
+{% for ipvpn_gw_remote_peer in switch.ipvpn_gateway.remote_peers | arista.avd.natural_sort %}
```

Comment:
```suggestion
{% for ipvpn_gw_remote_peer in switch.ipvpn_gateway.remote_peers | arista.avd.natural_sort("ip_address") %}
```

Reply:
Implemented suggestion in ibgp and ebgp neighbors template.